### ISSUE TITLE
Missing Module

### ISSUE THREAD

**Issue by Nathan3-14** [brentvollebregt/auto-py-to-exe#448]
# Interesting missing module error
## Error
I converted a very basic python project using tkinter and it gave the following error
![image](https://github.com/brentvollebregt/auto-py-to-exe/assets/117750247/7c6f5e21-28f8-42c7-ba5b-31449f20f99d)
And I was intereseted why it needed a module (ip address) despite it not being used in the code.

## Code
test.py:
```python
from tkinter import *

root = Tk()
test = Label(text="hi!")
test.pack()
root.mainloop()
```

**Comment by brentvollebregt** [comment]
This was fixed in PyInstaller 5.12 back in June - see more here: https://github.com/pyinstaller/pyinstaller/issues/7692

**Comment by Nathan3-14** [comment]
Cool thanks.
I updated my python and it seemed to work fine.
